src/ActionModal/utils/ActionManager.js

```diff
@@ -1,7 +1,7 @@
 import config from "src/../config"
 import { getSigner } from "./signer"
 import transaction from "./transactionTypes"
-import { uatoms } from "scripts/num"
```

Comment:
is uatoms still used?

Reply:
no, it was only being used for that particular function (converting the gas fee)

Reply:
then let'S remove the function?

Reply:
but yeah let's do it another PR

Reply:
this we will only be able to do once we have fully integrated the Transaction Abstraction.

For that we already have my PR #3407 waiting for the right time.